codeCoverageThresholdCheck.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 pushd ./src/ || exit
 
-THRESHOLD=50
+THRESHOLD=65
```

Comment:
We don't have a specific number in mind.  It will depend on how much code we have that is just pointing to an external library that we wouldn't want to test.